testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -607,15 +606,17 @@ def reference(self):
         return u
 
     def test_iterative_average(self, mobile, reference):
-        res = align.iterative_average(mobile, niter=1)
-        res = align.iterative_average(mobile, reference, niter=1, eps=0)
+        res = align.iterative_average(mobile, niter=10)
         res = align.iterative_average(mobile, reference, eps=1e-5)
         res = align.iterative_average(mobile, reference, select='bynum 1:10',
                                       weights=np.ones(10))
```

Comment:
These are just running through code paths and they do not assert anything. These are only marginally useful tests.

Please split `test_iterative_average` into multiple testing methods where each method tests one thing (eg default values, different eps, weights, ...) and where you 
1. run iterative_average() _once_
2. assert that the output is as expected

This will be more code but in this way we will see exactly if one specific part of the code fails.